gateway/tests/fixtures/fixtures.json

```diff
@@ -121,7 +125,7 @@
       "pk": 100,
       "fields": {
         "name": "runner",
-	"permissions": [61]
+	      "permissions": [61]
```

Comment:
is this indent needed?

Reply:
Mmm... This is weird. In my IDE I see it like this:

<img width="248" alt="Screenshot 2024-05-23 at 9 49 20 AM" src="https://github.com/Qiskit/qiskit-serverless/assets/9059044/6d1277ac-d5a8-46e8-9c24-548fe535d680">

I don't know why it appears like that in GitHub 🤔